packages/biometric-ed25519/src/utils.ts

```diff
@@ -90,4 +90,28 @@ export const recoverPublicKey = async (r, s, message, recovery) => {
 export const uint8ArrayToBigInt = (uint8Array: Uint8Array) => {
     const array = Array.from(uint8Array);
     return BigInt('0x' + array.map(byte => byte.toString(16).padStart(2, '0')).join(''));
+};
+
+// This function is design to convert any existing Uint8Array properties inside AuthenticatorAssertionResponse to ArrayBuffer
+export const convertUint8ArrayToBuffer = (obj) => {
+    // Function to handle the conversion of each property
+    function handleConversion(item) {
+        if (item instanceof Uint8Array) {
+            // Convert Uint8Array to ArrayBuffer
+            return item.buffer.slice(item.byteOffset, item.byteOffset + item.byteLength);
+        } else if (Array.isArray(item)) {
+            // Process array items
+            return item.map(handleConversion);
+        } else if (item !== null && typeof item === 'object') {
+            // Process object properties
+            return Object.keys(item).reduce((acc, key) => {
+                acc[key] = handleConversion(item[key]);
+                return acc;
+            }, {});
+        }
+        // Return the item if it's not a Uint8Array or an Object
+        return item;
+    }
+
+    return handleConversion(obj);
```

Comment:
Is `handleConversion` function necessary? I think it's better to just unwrap it if it's used in this function only.